openwisp_utils/admin_theme/templates/admin/base.html

```diff
@@ -17,6 +17,60 @@
     {% if LANGUAGE_BIDI %}<link rel="stylesheet" type="text/css" href="{% static "admin/css/responsive_rtl.css" %}">{% endif %}
 {% endblock %}
 {% block blockbots %}<meta name="robots" content="NONE,NOARCHIVE">{% endblock %}
+<style>
```

Comment:
Normally, I would say adding styles in html file is not ideal, but since these styles are relevant only as long as this file is around, it looks good to me! :smile: